main.css

```diff
@@ -26,7 +26,8 @@ Digital Dark Style by Michael Fabian 'Xaymar' Dirks <info@xaymar.com>.
 Layer-0,
 /* Background */ body,
 div.tabbed-pane-header,
-div.tabbed-pane-nav {
+div.tabbed-pane-nav,
+div#content {

```

Comment:
Trello applies the background to div#classic-body, not div#content. Please change this.

Reply:
You are right, i did not find that while searching earlier.